src/mjolnir/valhalla_ingest_transit.cc

```diff
@@ -0,0 +1,69 @@
+#include "baldr/rapidjson_utils.h"
+#include "mjolnir/ingest_transit.h"
+
+#include "config.h"
+#include <cxxopts.hpp>
+
+filesystem::path config_file_path;
+
+bool ParseArguments(int argc, char* argv[]) {
+  try {
+    // clang-format off
+    cxxopts::Options options(
+      "valhalla_ingest_transit",
+      "valhalla_ingest_transit " VALHALLA_VERSION "\n\n"
+      "valhalla_ingest_transit is a program that reads GTFS data. It converts a directory of transit feeds into protobuf tiles."
+      "\n\n");
+
+    options.add_options()
+      ("h,help", "Print this help message.")
+      ("v,version", "Print the version of this software.")
+      ("c,config", "Path to the json configuration file.", cxxopts::value<std::string>());
```

Comment:
it's good to have it not positional, more in line with most of valhalla's executables

Reply:
Is it possible to implement non-positional without flags, as the command line for `valhalla_convert_transit` is `valhalla_config [target_directory] [test_file]`. `valhalla_ingest_transit` looks already non positional?

Reply:
we can leave `valhalla_convert_transit` for the next PR IMO.

we can then take the opportunity to change that a bit and make it more consistent with the other exes (`config` as optional, even though required, and `test_file` as positional). to clarify, every argument that's not named is called "positional", so the previous behavior was 2 positional arguments. 

`implement non-positional without flags`, not sure what that means :sweat_smile: